nondex-test/src/test/java/edu/illinois/nondex/core/AbstractCollectionTest.java

```diff
@@ -56,16 +58,22 @@ protected void assertParameterized(T ds, Object derived, String str) {
         }
     }
 
-    protected void assertEqualstUnordered(String msg, String expected, String actual) {
-        Assert.assertEquals(msg + ": " + expected + " =/= " + actual, expected.length(), actual.length());
-        String trimmed = expected.substring(1, expected.length() - 1);
+    private String[] trimAndSplitStrings(String msg) {
+        String trimmed = msg.substring(1, msg.length() - 1);
         String[] elems = trimmed.split(",");
```

Comment:
Can it `split(", ")` and then not even require the loop for `trim()`?

Reply:
yes that should work. Will update the PR